src/Compilers/Core/Portable/InternalUtilities/Hash.cs

```diff
@@ -171,6 +171,7 @@ internal static int GetFNVHashCode(byte[] data)
             return hashCode;
         }
 
+#if !CODE_STYLE
```

Comment:
Let's not complicate this file. If you can't include this file anymore then your code really shouldn't be sharing with the master compiler layer. Just copy the code you need.

Reply:
> Just copy the code you need.

I went to _great lengths_ to not have this pull request result in duplicated code. Requesting an exception for this case, which can be reviewed in the future if it looks like it will become more complicated.

Reply:
Second alternative proposal: keep this as-is for now, and file an immediate follow-up pull request to remove the use of `Hash` from the formatter implementation, revert the change to this file, and unlink it

Reply:
As long as the link is removed, fine with me.

Reply:
➡️ #31404